packages/ez-react/src/recipes/area/AreaChart.stories.tsx

```diff
@@ -52,3 +61,17 @@ Default.args = {
   },
   data: evolutionData,
 };
+
+Default.args = defaultArguments;
+
+export const MultiArea = MultiAreaTemplate.bind({});
+
+MultiArea.args = {
+  ...defaultArguments,
+  area: { ...defaultArguments.area, opacity: 0.5 },
+  yAxis: {
+    domainKeys: ['yValue2', 'yValue1', 'yValue'],
```

Comment:
```suggestion
    domainKeys: ['yValue', 'yValue1', 'yValue2'],
```

Reply:
Did this on purpose so that the `yValue2` domain key is rendered first. Since we don't sort the domain keys.